### DISCUSSION TITLE
Is it possible to set the header / hero image for an album?

### DISCUSSION THREAD

**Discussion by joffotron** [LycheeOrg/Lychee#2145] [Q&A] [answered]
(I'm on v5)

I see we can right-click an image to set it as the album cover - is it possible to mark an image to permanentely set it at the header/hero image for an album, rather than the current "pick a random image" behaviour?

**Comment by ildyria** [comment]
Already planned. 😁

⁣Get BlueMail for Android ​

**Comment by ildyria** [comment, marked as answer]
implemented.
